applications/milliwatt/src/milliwatt_listener.erl

```diff
@@ -26,10 +26,7 @@
 -record(state, {}).
 -type state() :: #state{}.
 
--define(BINDINGS, [{'route', [{'types', ?RESOURCE_TYPES_HANDLED}
-                             ,{'restrict_to', ['no_account']}
```

Comment:
1. it was like that before
2. there is a use case in make-busy for that
3. there is no easy way to make make-busy do calls from devices unknown to kazoo
4. karl says so